### PR TITLE
Update help window for updated usage of edit command

### LINKED ISSUE #73: Update help shown to user with new syntax for add/remove tag
With the new delete tag prefix `dt/` for the edit command, the help command/window should show this as a possible syntax as well.

### PR DESCRIPTION
**Pull request by sayomaki**
Resolves #73

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -19,10 +19,11 @@ public class HelpWindow extends UiPart<Stage> {
...
     private static final String HELP_EDIT_COMMAND = "Edit Command \n"
-            + "- Format: edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG] [r/REMARK]\n"
-            + "- Example: edit 1 p/87654321"
+            + "- Format: edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG] [dt/TAG] [r/REMARK]\n"
+            + "- Example: edit 1 p/87654321 t/Tutor dt/Student"
             + "- INDEX is the index of the customer displayed. Items in the [square brackets] are optional "
-            + "- and can be input in any order!";
+            + "- and can be input in any order! TAG can be both a tag to be added or removed, depending on "
+            + "- the prefix for the item.";
     private static final String HELP_DELETE_COMMAND = "Delete Command\n"
```
Comment (Tkaixiang) [inline src/main/java/seedu/address/ui/HelpWindow.java:27]: You might want to consider remove the `-` since it is part of the same point

**Review by Tkaixiang — APPROVED** [review]
pre-approved pending minor change above ^

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (Vatinius) [reply src/main/java/seedu/address/ui/HelpWindow.java:27]: On a similar note, i've noticed something similar at line 53 of HelpWindow.java. It results in a weird-looking text format (see screenshot below). Maybe you could look into resolving that too with this PR?
image

If not, I can create another PR just to resolve it.

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -50,7 +51,7 @@ public class HelpWindow extends UiPart<Stage> {
             + "- Format: exit\n"
             + "- Example: exit";
     private static final String HELP_MORE_INFORMATION = "For more detailed information, "
-            + "- visit the PROperty User Guide at: https://ay2425s1-cs2103t-f15-3.github.io/tp/UserGuide.html";
+            + " visit the PROperty User Guide at: https://ay2425s1-cs2103t-f15-3.github.io/tp/UserGuide.html";
```
Comment (Vatinius) [inline src/main/java/seedu/address/ui/HelpWindow.java:54]: There is an extra whitespace at the beginning of the sentence! You might want to remove it before merging.

**Comment by Vatinius** [comment]
lgtm!

**Pull request merged**